ansible/roles/esdc-mgmt/templates/esdc/es-post-deploy.sh.j2

```diff
@@ -23,6 +23,18 @@ esdc_pre_boot() {
 	log "$(sudo -u erigones ${CTL} pre_boot 2>&1)"
 }
 
+log "Populating authorized_keys"
```

Comment:
Please add a comment with a simple explanation why we are doing this.

(A link to `https://github.com/erigones/esdc-factory/issues/43` should be enough)